diff -ur geos-3.2.0rc3/tests/xmltester/tinyxml/tinyxml.h geos-3.2.0rc3.new/tests/xmltester/tinyxml/tinyxml.h
--- geos-3.2.0rc3/tests/xmltester/tinyxml/tinyxml.h	2009-09-28 12:35:23.000000000 +0200
+++ geos-3.2.0rc3.new/tests/xmltester/tinyxml/tinyxml.h	2009-12-05 12:04:27.781250000 +0100
@@ -57,7 +57,7 @@
 // new safe versions. This probably doesn't fully address the problem,
 // but it gets closer. There are too many compilers for me to fully
 // test. If you get compilation troubles, undefine TIXML_SAFE
-#define TIXML_SAFE
+// #define TIXML_SAFE
 
 #ifdef TIXML_SAFE
 	#if defined(_MSC_VER) && (_MSC_VER >= 1400 )
